requestor-tutorials/vm-runtime/golem-certificates.md

```diff
@@ -0,0 +1,80 @@
+# Golem-certificates
+
+## Installing cli tool
+
+Prerequisites:
+
+- [Rust toolchain](https://rustup.rs/)
+- Git
+
+```bash
+git clone https://github.com/golemfactory/golem-certificate
+cargo install --path golem-certificate/cli
+```
+
+After that, in other terminal enter UI tool by typing:
+
+```bash
+golem-certificate-cli ui
+```
+
+## Generating key-pair
+
+Create new key-pair in `Create key pair` option in main menu and name files appropriately.
+
+## Generating root certificate
+
+Prerequisites:
+
+- key-pair for your root certificate
+
+Navigate to `Create certificate` option.
+After that, put your propertiies as needed (Key usage, Validity period, Permissions and subject information) and enter your generated **public** key.
```

Comment:
Here it would be worth mentioning that `Key usage` must include `sign certificate`.